.github/workflows/deploy-code.yml

```
@@ -28,12 +31,12 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Install deps
+        run: pip install -U pip setuptools virtualenv wheel
+      - name: Build sdist
+        run: python3 setup.py sdist bdist_wheel
```

The future proof way to do this is to use:

```suggestion
        run: pip install -U pip setuptools build
      - name: Build sdist
        run: python3 -m build .
```

but for now this works fine.

---

Ok, sounds good for the future.